modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Jetpack_Simple_Payments_Widget' ) ) {
+	/**
+	 * Simple Payments Button
+	 *
+	 * Display a Simple Payment Button as a Widget.
+	 */
+	class Jetpack_Simple_Payments_Widget extends WP_Widget {
+		/**
+		 * Constructor.
+		 */
+		function __construct() {
+			$widget = array(
+				'classname' => 'simple-payments',
+				'description' => __( 'Add a Simple Payment Button as a Widget.', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			);
+
+			parent::__construct( 'Jetpack_Simple_Payments_Widget', __( 'Simple Payments', 'jetpack' ), $widget );
+		}
+
+		/**
+		 * Front-end display of widget.
+		 *
+		 * @see WP_Widget::widget()
+		 *
+		 * @param array $args     Widget arguments.
+		 * @param array $instance Saved values from database.
+		 */
+		function widget( $args, $instance ) {
+			echo $args['before_widget'];
+
+			$title = apply_filters( 'widget_title', $instance['title'] );
+			if ( ! empty( $title ) ) {
+				echo $args['before_title'] . $title . $args['after_title'];
+			}
+
+			echo '<div class="simple-payments-content">';
+
+			if( ! empty( $instance['product_post_id'] ) ) {
+				$attrs = array( 'id' => $instance['product_post_id'] );
+			} else {
+				$product_posts = get_posts( [
```

Comment:
This `[]` notation is not going to be okay in Jetpack, since we support PHP 5.2+.

Reply:
Updated on 0b74235429d74625242a9ea780fd5077e4feb711.